src/connectors/github_organizations.py

```diff
@@ -0,0 +1,341 @@
+    pipe = f'data.{base_name}_PIPE'
+    landing_table = f'data.{base_name}_CONNECTION'
+
+    DATE_REGEXP = r'.+(\d{4})(\d{2})(\d{2})T(\d{2})(\d{2})(\d{2})Z.*'.replace("\\", "\\\\")
+    DATE_ISO8601_BACKREFERENCES = r'\1-\2-\3T\4:\5:\6Z'.replace("\\", "\\\\")
+
+    config_ingest_task = f'''
+INSERT INTO {landing_table} (
+    _file, _line, _modified, ref, before, after, created, deleted, forced, base_ref, compare, commits, head_commit,
+    repository, pusher, organization, sender, _fivetran_synced, action, check_run, check_suite, number, pull_request,
+    label, requested_team, ref_type, master_branch, description, pusher_type, review, changes, comment, issue, id, sha, name, target_url,
+    context, state, commit, branches, created_at, updated_at, assignee, release, membership, alert, scope, member, requested_reviewer, team,
+    starred_at, pages, project_card, build, deployment_status, deployment, forkee, milestone, key, project_column, status, avatar_url
+)
+SELECT CURRENT_TIMESTAMP() insert_time
+    , value raw
+    , HASH(value) hash_raw
+    , value:ref::VARCHAR(256) ref
+    , value:before::VARCHAR(256) before
+    , value:after::VARCHAR(256) after
+    , value:created::BOOLEAN created
+    , value:deleted::BOOLEAN deleted
+    , value:forced::BOOLEAN forced
+    , value:base_ref::VARCHAR(256) base_ref
+    , value:compare::VARCHAR(256) compare
+    , value:commits::VARIANT commits
+    , value:head_commit::VARIANT head_commit
+    , value:repository::VARIANT repository
+    , value:organization::VARIANT organization
+    , value:sender::VARIANT sender
+    , value:action::VARCHAR(256) action
+    , value:check_run::VARIANT check_run
+    , value:check_suite::VARIANT check_suite
+    , value:number::'NUMBER(38,0)' number
+    , value:pull_requests::VARIANT pull_request
+    , value:labels_url::VARIANT label
+    , value:requested_teams::VARIANT requested_team
+    , value:ref_type::VARCHAR(256) ref_type
+    , value:master_branch::VARCHAR(256) master_branch
+    , value:description::VARCHAR(256) description
+    , value:pusher_type::VARCHAR(256) pusher_type
+    , value:review_comments::VARIANT review
+    , value:changed_files::VARIANT changes
+    , value:comments_url::VARIANT comment
+    , value:issues_url::VARIANT issue
+    , value:id::NUMBER(38,0) id
+    , value:head_sha::VARCHAR(256) sha
+    , value:name::VARCHAR(256) name
+    , value:target_url::VARCHAR(8192) target_url
+    , value:context:: VARCHAR(256) context
+    , value:state:: VARCHAR(256) state
+    , value:commits_url::VARIANT commit
+    , value:branches_url:VARIANT branches
+    , value:created_at::TIMESTAMP_LTZ(9) created_at
+    , value:updated_at::TIMESTAMP_LTZ(9) updated_at
+    , value:assignees_url::VARIANT assignee
+    , value:releases_url::VARIANT release
+    , value:members_url::VARIANT membership
+    , value:FILL IN ALERT
+    , value:FILL IN SCOPE
+    , value:members:VARIANT member
+    , value:requested_reviewers::VARIANT requested_reviewer
```

Comment:
`requested_reviewers` vs `requested_reviewer` something might be off there

Reply:
I checked the json file again and it says requested_reviewers